hadoop-hdfs-project/hadoop-hdfs-rbf/src/test/java/org/apache/hadoop/hdfs/server/federation/router/TestRouterTrash.java

```diff
@@ -282,6 +282,13 @@ public void testMultipleMountPoint() throws IOException,
     fileStatuses = fs.listStatus(new Path("/user/test-trash/.Trash/Current/" + MOUNT_POINT2));
     assertEquals(0, fileStatuses.length);
 
+    // In ns1, make a trash path with timestamp to simulate a trash path.
+    String trashPath = "/user/test-trash/.Trash/" + System.currentTimeMillis();
+    client1.mkdirs(trashPath, new FsPermission("770"),
+        true);
+    fileStatuses = fs.listStatus(new Path("/user/test-trash/.Trash"));
```

Comment:
Can we make the tests a little more exhaustive?

Reply:
Thanks for reivew.  For this it is enough and others will be more complicated.